src/repository/userDAO.js

```diff
@@ -1,17 +1,17 @@
 const { PutCommand, QueryCommand, DeleteCommand, UpdateCommand, GetCommand } = require('@aws-sdk/lib-dynamodb');
 const { TableName, UsernameIndex, CLASS_USER, runCommand } = require('../utilities/dynamoUtilities');
 
-async function putUser(Item) {
+const putUser = async (Item) => {
```

Comment:
I was doing function initially, then started changing it to const. Since I've already changed everything to use const, I'd say we just keep it that way if we do add any more endpoints.